examples/example_uq_bootstrap.py

```diff
@@ -0,0 +1,133 @@
+"""
+.. _tut_mcmc:
+
+Bootstrapping
+=============
+
+In this example, we demonstrate how to perform uncertainty quantification (UQ) using
+bootstrap method. We use a Stillinger-Weber (SW) potential for silicon that is archived
+in OpenKIM_.
+
+For simplicity, we only set the energy-scaling parameters, i.e., ``A`` and ``lambda`` as
+the tunable parameters. Furthermore, these parameters are physically constrained to be
+positive, thus we will work in log parameterization, i.e. ``log(A)`` and ``log(lambda)``.
+These parameters will be calibrated to energies and forces of a small dataset,
+consisting of 4 compressed and stretched configurations of diamond silicon structure.
+"""
+
+
+##########################################################################################
+# To start, let's first install the SW model::
+#
+#    $ kim-api-collections-management install user SW_StillingerWeber_1985_Si__MO_405512056662_006
+#
+# .. seealso::
+#    This installs the model and its driver into the ``User Collection``. See
+#    :ref:`install_model` for more information about installing KIM models.
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from kliff.calculators import Calculator
+from kliff.dataset import Dataset
+from kliff.dataset.weight import MagnitudeInverseWeight
+from kliff.loss import Loss
+from kliff.models import KIMModel
+from kliff.models.parameter_transform import LogParameterTransform
+from kliff.uq.bootstrap import BootstrapEmpiricalModel
+from kliff.utils import download_dataset
+
+##########################################################################################
+# Before running bootstrap, we need to define a loss function and train the model. More
+# detail information about this step can be found in :ref:`tut_kim_sw` and
+# :ref:`tut_params_transform`.
+
+# Instantiate a transformation class to do the log parameter transform
+param_names = ["A", "lambda"]
+params_transform = LogParameterTransform(param_names)
```

Comment:
Let's remove the `params_transform`. This adds additional barriers, and it is not the point of this tutorial.